index.html

```diff
@@ -850,19 +851,14 @@
             </li>
             <li>Let <em>P</em> be a new <a>Promise</a>.
             </li>
-            <li>Return <em>P</em>.
+            <li>Return <em>P</em>, but continue running these steps in
+            parallel.
             </li>
-            <li>Run the following steps in parallel:
-              <ol>
-                <li>
-                  <a>Monitor the list of available presentation displays</a>.
-                </li>
-                <li>
-                  <a>Queue a task</a> <em>T</em> to request user permission for
-                  the use of a <a>presentation display</a> and selection of one
-                  presentation display.
-                </li>
-              </ol>
+            <li>Run the steps to <a>monitor the list of available presentation
```

Comment:
Should clarify that this only needs to be started if the UA was not already monitoring the list of displays.